<launch>

	<!--  无人机飞行中的期望高度 -->
        <arg name="desire_z"  type="double" value="1.5"/>
        <arg name="change" type="double" value="0.02" />
        <!--                                       持续向飞控发送点                                 -->
        <node  name = "send_info_node" pkg = "offb" type = "send_info_node" output = "screen"> </node>
        <!--                                       主要的控制程序                                   -->
        <node  name = "node_flyput_node" pkg = "offb" type = "node_flyput_node" output = "screen"  launch-prefix="gnome-terminal --window --">
                <param name="desire_z" value="$(arg desire_z)"/>
                <param name="change" value="$(arg change)" />
                </node>
        <!--                                       显示无人机的状态以及速度                	    -->
        <node  name = "showinfo_node" pkg = "offb" type = "showinfo_node" output = "screen" launch-prefix="gnome-terminal --window --"> </node>
        <!--                                       改变无人机的模式                                  -->
        <node  name = "set_mode_node" pkg = "offb" type = "set_mode_node" output = "screen" launch-prefix="gnome-terminal --window --"> </node>

	<!--                                       串口输入改变舵机状态                              -->
        <node name="servo_control_node"  pkg = "offb"  type = "servo_control_node" output = "screen"  launch-prefix="gnome-terminal --window --"> </node>
	<!--                                      打开电脑自带摄像头                                                  -->
	<node pkg="offb" type="cam.py" name="cam_node" respawn="false" />
	<!--                                      识别圆心并存储坐标                                                  -->
	<node pkg="offb" type="cycle_detect_new_node" name="cycle_detect_new_node" respawn="false" launch-prefix="gnome-terminal --window --"/>
</launch>
